tests/checks/complete.fish

```diff
@@ -9,6 +9,8 @@ complete -c complete_test_alpha3 --no-files -w 'complete_test_alpha2 extra2'
 
 complete -C'complete_test_alpha1 arg1 '
 # CHECK: complete_test_alpha1 arg1
+complete --unescaped -C'complete_test_alpha1 arg1 '
+# CHECK: complete_test_alpha1\ arg1\
```

Comment:
The `--unescaped` option makes the output escaped, so it should be called `--escape` (I think that's better than `--escaped`)